gravity.sh

```diff
@@ -29,6 +29,7 @@ EOM
 
 adListFile=/etc/pihole/adlists.list
 adListDefault=/etc/pihole/adlists.default
```

Comment:
This is interesting, I haven't changed that. This line goes back to a commit from Adam in January 352ebc3193d17ae4e075eec7f71bfe909ab8fa06